components/03-modules/gallery/mocks/gallery.json

```diff
@@ -0,0 +1,52 @@
+{
+  "class": "",
+  "typeHorizontal": "false",
+  "imageMain": {
+    "dataSrc": "/images/gallery/product-img-496-496.jpg",
+    "alt": "Sample alt"
+  },
+  "thumbs": [
+    {
+      "imageThumb": {
+        "dataSrc": "/images/gallery/product-img-80-80.jpg",
+        "active": "true",
+        "alt": ""
+      }
+    },
+    {
+      "imageThumb": {
+        "dataSrc": "/images/gallery/product-img-80-80.jpg",
+        "alt": ""
+      }
+    },
+    {
+      "imageThumb": {
+        "dataSrc": "/images/gallery/product-img-80-80.jpg",
+        "alt": ""
+      }
+    },
+    {
+      "imageThumb": {
+        "dataSrc": "/images/gallery/product-img-80-80.jpg",
+        "alt": ""
+      }
+    }
+  ],
+  "variants": [
```

Comment:
Do not use the same config files as in Fractal, it's not so readable. If you need two separate variants, create more mock files.